release/rancher/rancher.go

```diff
@@ -0,0 +1,80 @@
+package rancher
+
+import (
+	"bufio"
+	"errors"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	rancherImagesBaseURL  = "https://github.com/rancher/rancher/releases/download/"
+	rancherImagesFileName = "/rancher-images.txt"
+)
+
+func ListRancherImagesRC(tag string) (string, error) {
+	downloadURL := rancherImagesBaseURL + tag + rancherImagesFileName
+	imagesFile, err := rancherImages(downloadURL)
+	if err != nil {
+		return "", err
+	}
+	rcImages, err := findRCNonMirroredImages(imagesFile)
+	if err != nil {
+		return "", err
+	}
+
+	if len(rcImages) == 0 {
+		return "There are none non-mirrored images still in rc form for tag " + tag, nil
+	}
+
+	output := "The following non-mirrored images for tag *" + tag + "* are still in RC form\n```\n"
+	for _, image := range rcImages {
+		output += image + "\n"
+	}
+	output += "```"
+
+	return output, nil
+}
+
+func findRCNonMirroredImages(images string) ([]string, error) {
```

Comment:
I think we can change the function signature to remove the returned error since it doesn't look like we ever have the potential to return one.